src/main/resources/assets/js/components/NewAppModalComponent.jsx

```diff
@@ -259,9 +287,105 @@ var NewAppModalComponent = React.createClass({
                 errors={errors}>
               <input />
             </FormGroupComponent>
+            <hr />
+            <h4>Hide docker container settings</h4>
+            <FormGroupComponent
+                attribute="image"
+                help='Image for docker'
+                label="Image"
+                model={model}
+                errors={errors}>
+              <input />
+            </FormGroupComponent>
+            <FormGroupComponent
+                attribute="network"
+                help='Network host'
+                label="Network"
+                model={model}
+                errors={errors}>
+              <select>
+                <option value="host">Host</option>
+                <option value="host_two">Host two</option>
+              </select>
+            </FormGroupComponent>
+            <hr />
+            <h4>Privileges</h4>
+            <FormGroupComponent
+                attribute="privileges"
+                help='Select to give this container access to all devices on the host'
+                label="Extend runtime privileges to this container"
+                model={model}
+                errors={errors}>
+              <input type="checkbox" value="true"/>
+            </FormGroupComponent>
+            <hr />
+            <h4>Parameters</h4>
+            <FormGroupComponent
+                attribute="key"
+                label="Key"
+                model={model}
+                errors={errors}>
+              <input />
+            </FormGroupComponent>
+            <FormGroupComponent
+                attribute="value"
+                label="Value"
+                model={model}
+                errors={errors}>
+              <input />
+            </FormGroupComponent>
+            {inputs.map(function (result) {
```

Comment:
What is this doing? No value field there..

And you should create a function from it to create the list directly from state.parameterCount.